lms/templates/certificates/_accomplishment-banner.html

```diff
@@ -62,6 +73,21 @@ <h3 class="sr-only">${_("Print or share your certificate:")}</h3>
                             <span class="action-label">${_("Post on Facebook")}</span>
                         </a>
                       %endif
+                      % if share_settings.get('CERTIFICATE_TWITTER', False):
+                      <% twitter_url = 'https://twitter.com/intent/tweet?text=' + twitter_share_text + '&url='+ share_url%>
+                        <a
+                          data-tooltip="${_('Share on Twitter')}"
+                          class="action action-share-twitter btn btn-overlay btn-small icon-only"
+                          aria-haspopup="true"
+                          aria-expanded="false"
+                          href="${twitter_url}"
+                          target="_blank"
+                          title="${_('Share on Twitter')}"
+                          onclick="popupWindow('${twitter_url}', 'tweetWindow', 640, 480); return false;">
+                          <span class="action-label">${_("Tweet this Accomplishment")}</span>
+                          <i class="icon fa fa-twitter" aria-hidden="true"></i>
+                        </a>
+                      %endif
                     %endif
 
                     %if linked_in_url:
```

Comment:
@cptvitamin linkedin is taking the user to another location. What you suggest if we keep this as an anchor instead of button ?